Index: src/test/java/com/library2/runner/CukesRunner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.library2.runner;\r\n\r\nimport io.cucumber.junit.Cucumber;\r\nimport io.cucumber.junit.CucumberOptions;\r\nimport org.junit.runner.RunWith;\r\n\r\n    @RunWith(Cucumber.class)\r\n    @CucumberOptions(\r\n            plugin = {\r\n                    \"pretty\" ,\r\n                    \"html:target/cucumber-report.html\",\r\n                    \"json:target/cucumber.json\",\r\n                    \"rerun:target/rerun.txt\" ,\r\n                    \"me.jvt.cucumber.report.PrettyReports:target/cucumber\"\r\n            } ,\r\n            features = \"src/test/resources/features\" , // from content\r\n            glue = \"com/library2/step_definitions\" , //from source root\r\n\r\n            dryRun = false,\r\n            tags = \"@smoke\"\r\n               ,\r\n                publish = true\r\n    )\r\n\r\n    public class CukesRunner {\r\n    }\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/com/library2/runner/CukesRunner.java b/src/test/java/com/library2/runner/CukesRunner.java
--- a/src/test/java/com/library2/runner/CukesRunner.java	
+++ b/src/test/java/com/library2/runner/CukesRunner.java	
@@ -17,7 +17,7 @@
             glue = "com/library2/step_definitions" , //from source root
 
             dryRun = false,
-            tags = "@smoke"
+            tags = "@us001_t06"
                ,
                 publish = true
     )
